projectile.el

```diff
@@ -1949,7 +1970,9 @@ regular expression."
 (defun projectile-regenerate-tags ()
   "Regenerate the project's [e|g]tags."
   (interactive)
-  (if (boundp 'ggtags-mode)
+  (if (and (boundp 'ggtags-mode)
+        (or (eq projectile-tags-backend 'auto)
```

Comment:
The indentation of this expression is incorrect. You can simplify it a bit by using `memq`.